### PR TITLE
Matching colours of the DARK VARIANT more closely with the LIGHT VARIANT

### PR DESCRIPTION
**Pull request by wimstefan**
This is an approach to match the colours of the DARK VARIANT more closely with 
the LIGHT VARIANT. Furthermore I changed the colours for search, diff, folds and 
cursor{column,line} to make it more readable. And I corrected some colour values 
where the RGB values didn't match the HEX values.

Of course you don't have to accept that changes but nevertheless it would be 
nice if you'd tell me what you like and more importantly what you don't like.

Perhaps I can make the changes more acceptable ... ;-)

### CODE REVIEW

**Comment by NLKNguyen** [comment]
Wow, that's a big change. The colors reminds me of Atom's default dark theme, which is nice and pleasant. The red color seems a bit too dark. It's great that you noticed and corrected the mismatch HEX and 256 numbers.

Over all, I think it's a good start for an alternative color scheme. Please hold onto that for sometime. I've started a revision to enable arbitrary many different color palettes to co-exist on PaperColor. That way, your color changes can stand as its own color palette and be next to the current Light and Dark.

It's gonna take me sometime because I'm also making support for 16-color. But don't worry, it won't be a limitation for 256-color or true-color palettes. I'm gonna put a roadmap on README at some point.

In short, thanks for your PR and please hold on these changes until after the revision when there will be a different way of color organizing.

**Comment by wimstefan** [comment]
Thanks for that informative response! It's quite exciting to hear what's in front of us. Can't wait to see that :smiley: 

And I'm quite happy to hear that you no completely dislike my changes - it's quite a challenge to get the colours right on a dark background ... certainly the diff colours. I'll keep working on that to improve it and make it as readable as possible!

In the meantime I noticed that there are quite some incoherences between the HEX and RGB colours. I'm willing to correct them all but I first want to hear from you which value you prefer: the HEX or the RGB (256) colour?

**Comment by wimstefan** [comment]
Regardless of the exciting changes to come I tweaked the colours a bit more to follow up your advice. And I started to correct the incoherences between HEX and RGB colours ... for now only in the DARK VARIANT.

**Comment by NLKNguyen** [comment]
This is the roadmap to keep track of the progress https://github.com/NLKNguyen/papercolor-theme/blob/master/ROADMAP.md
When 1.0 is released, alternative color palettes will be possible. Regarding the preference for HEX or 256-color, I prefer 256-color, and the HEX value should be the equivalence of the 256-color value. The reason is because 256-color is the bottle neck of the two. By the way, I don't know how to call correctly the 256-color type. RGB, on the other hand, seems more like this thing `(255, 255, 255)`
Hah, I see you noticed the `s:wine` color :smile: . It was added way after the color palettes were established. I mainly use it for noisy syntax that is more like distraction so that it will be less visible than the more important part of the code. I may just drop it in the future and use the existing `s:purple` instead.
In any case, the changes can only be added in the future code base, not the current one, so stay tuned :+1:

**Comment by NLKNguyen** [comment]
I close this for now. My plan for making the color theme a framework is still progressing. I estimate by this summer, it will be possible to derive theme from PaperColor. I'll post a comment here once it's available.
